### ISSUE TITLE
Support the NEOS MA-20 (other VDP) in the debugger

### ISSUE THREAD

**Issue by sdsnatcher** [openMSX/openMSX#2013]
The debugger has problems to run with the NEOS MA-20. Maybe it's hardcoded to the default I/O ports?

For example, with the MA-20 connected:
- The Bitmapped VRAM View "Current Page" combo box doesn't work
- The TIle VRAM View shows incorrect info about the video mode: all data shown are from the internal VDP
- The VDP registers view seems to be showing register data form the internal VDP

**Comment by m9710797** [comment]
I didn't check the code, but it's highly likely that the debugger indeed only interacts with the primary (builtin) VDP. So this bug report is 100% valid.

On the other hand, is this really an important issue? I mean were you really trying to debug something on the NEOS MA-20, or were you only thinking about rare scenarios that might not yet be supported? (And the latter is good of course, thanks for reporting such things!)

IMHO it isn't needed that the GUI supports really everything that the openMSX core can handle. Especially not when a *very* rare use case will complicate the UI for +99.9% of the users who won't ever need to select the VDP. (Ok, we could hide the selection when there's only one VDP present).

Patches welcome. But for me personally this is a very low priority item.

**Comment by sdsnatcher** [comment]
Yes, I'm trying to debug something on the NEOS MA-20.   It's not just a rare scenario not yet supported.

**Comment by MBilderbeek** [comment]
@m9710797 is this still valid for the new debugger? I guess it is, but please confirm. Then we can transfer it to the main issue tracker.

**Comment by m9710797** [comment]
It's still valid in the new debugger. But personally I still think we shouldn't spend time on this (as in: there are many other more important items).

Since it's unlikely we'll ever implement this (unless a volunteer steps up), it may even be better to not transfer this ticket?

**Comment by MBilderbeek** [comment]
I'd still like to transfer it. It is a rare case, but valid. This hardware exists.
